test/Microsoft.AspNetCore.Mvc.Core.Test/Formatters/StringOutputFormatterTests.cs

```diff
@@ -12,22 +12,31 @@
 
 namespace Microsoft.AspNetCore.Mvc.Formatters
 {
-    public class TextPlainFormatterTests
+    public class StringOutputFormatterTests
     {
-        public static IEnumerable<object[]> OutputFormatterContextValues
+        public static IEnumerable<object[]> CanWriteResultForStringTypesData
         {
             get
             {
-                // object value, bool useDeclaredTypeAsString, bool expectedCanWriteResult
-                yield return new object[] { "valid value", true, true };
-                yield return new object[] { null, true, true };
-                yield return new object[] { null, false, false };
-                yield return new object[] { new object(), false, false };
+                // object value, bool useDeclaredTypeAsString
+                yield return new object[] { "declared and runtime type are same", true };
+                yield return new object[] { "declared and runtime type are different", false };
+                yield return new object[] { null, true };
+            }
+        }
+
+        public static IEnumerable<object[]> CannotWriteResultForNonStringTypesData
```

Comment:
Suggest `CanWriteStringsData` and `CannotWriteNonStringsData`. Same for the corresponding test names.